index.html

```diff
@@ -16,51 +16,88 @@
+                <ul id="nav-mobile" class="right hide-on-med-and-down">
+                    <li><a href="#">Personal Settings</a></li>
+                </ul>
+            </div>
+        </nav>
     </header>
+    <!-- semantic main Page Layout-->
     <main class="main-content">
-        <div class="searchBand">
-            <h3> Search for a Band: </h3>
-            <div class="input-group mb-3">
-                <input type="text" class="form-control" id="artist-input">
-                <div class="input-group-append">
-                    <input type="submit" id="searchBtn">
+        <div class="row">
+            <aside class="col s12 m4 l3 search-band">
+                <h3> Search for a Band: </h3>
+                <div class="input-group mb-3">
+                    <input type="text" class="form-control" id="artist-input">
+                    <div class="input-group-append">
+                        <input type="submit" id="searchBtn">
+                    </div>
                 </div>
-            </div>
+                <div class="band-details">
+                    <div id="band-details">
+                        <h2 id="artist-name"></h2>
+                        <img id="artist-image" src="#" alt="artist image">
+                    </div>
+                    <ul id="band-features">
+                        <li class="genre"> Genre: </li>
+                        <li class="topSong"> #1 Rated Song: </li>
+                        <ol class="hits"> Hits:
+                            <li> Hit #1</li>
+                            <li> Hit #2</li>
+                            <li> Hit #3</li>
+                            <li> Hit #4</li>
+                        </ol>
+                        <li class="spotifyURL"> Spotify URL: </li>
+                    </ul>
+                </div>
+            </aside>
+            <section class="col s12 m8 l9 main-body">
+                <div class="row">
+                    <div class="col s12 m6 l6">
+                        <div class="card bands-in-town-list">
+                            <div class="card-title">The bands in Town</div>
+                            <div class="card-content">
+                                <p></p>
+                            </div>
+                        </div>
+                        <div class="card bands-in-town-scroll">
+                            <div class="card-title">Possible carousel of bands</div>
+                            <div class="card-content">
+                                <p></p>
+                            </div>
+                        </div>
+                        <div id="blurb-about-site" class="card">
+                            <div class="card-title">Blurb about Site</div>
+                            <div class="card-content">
+                                <p> The content written will apear in here this is the dive here it goes in</p>
+                            </div>
+                        </div>
```

Comment:
I think having the blurb card first is better (on top of the bandsintown list and scroll)
```suggestion
                        <div class="card" id="blurb-about-site">
                            <div class="card-title">Blurb about Site</div>
                            <div class="card-content">
                                <p> The content written will appear in here this is the dive here it goes in</p>
                            </div>
                        </div>
                        <div class="card">
                            <div class="card-title">The bands in Town</div>
                            <div class="card-content bands-in-town-list"></div>
                        </div>
                        <div class="card">
                            <div class="card-title">Possible carousel of bands</div>
                            <div class="card-content bands-in-town-scroll"></div>
                        </div>
```